### ISSUE TITLE
mean files with /r/f

### ISSUE THREAD

**Issue by humbertocruz** [linnovate/mean#1378]
Hello, I recently install mean.io on OSX El Capitan and its generate a error when executed:

 env: node\r: No such file or directory

if I edit the files removing the windows linefeed it's works fine.

Thanks

**Comment by congling0251** [comment]
I saw this too in OSX EI Capitan when mean package [pkname].

**Comment by bnlong** [comment]
I also saw the same issue on Ubuntu 14.04. I solved the problem by installing an older version:  
npm install -g mean-cli@0.10.14

**Comment by bioub** [comment]
Same here on OSX, solved using dos2unix on /usr/local/lib/node_modules/mean-cli/bin : 
    brew install dos2unix
    find /usr/local/lib/node_modules/mean-cli/bin -type f -exec dos2unix {} \;

**Comment by rjVapes** [comment]
There was a line ending conflict without a .gitattributes file, so I added one and have updated the registry.  `npm update -g mean-cli` should fix your issues.  Feel free to re-open if not.

**Comment by jmbldwn** [comment]
Still happening....

**Comment by rjVapes** [comment]
Awesome, so it seems the issue is with actually doing an 'npm publish' from windows, not with the repository itself.  I bumped the version and published from OSX.  Seems to be working now (when I tested previously I npm installed from the repository directly pre-publish to test so missed the actual issue).  Should be good now.

**Comment by jmbldwn** [comment]
Seems to work, though I may have tainted my results with the prior fix...

On Oct 31, 2015, at 4:07 PM, John Webb <notifications@github.com<mailto:notifications@github.com>> wrote:


Awesome, so it seems the issue is with actually doing an 'npm publish' from windows, not with the repository itself. I bumped the version and published from OSX. Seems to be working now (when I tested previously I npm installed from the repository directly pre-publish to test so missed the actual issue). Should be good now.
